portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,202 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: 'Jost', sans-serif;
+    list-style: none;
+    text-decoration: smooth;
+}
```

Comment:
I recommend keeping the properties in a css class in alphabetical order. It makes it more readable. I won't strictly enforce it for SPS. But overall a good thing to follow,
box-sizing:
font-family:
list-style:
margin:
padding:
tetx-decoration:

Reply:
That's a very good point!